cmd/osm-controller/osm-controller.go

```diff
@@ -107,6 +107,7 @@ func init() {
 	flags.BoolVar(&optionalFeatures.WASMStats, "stats-wasm-experimental", false, "Enable a WebAssembly module that generates additional Envoy statistics")
 	flags.BoolVar(&optionalFeatures.EgressPolicy, "enable-egress-policy", false, "Enable OSM's Egress policy API")
 	flags.BoolVar(&optionalFeatures.MulticlusterMode, "enable-multicluster", false, "Enable multicluster mode in OSM")
+	flags.BoolVar(&optionalFeatures.SnapshotCache, "enable-snapshotcache", false, "Enable snapshot cache XDS mode in OSM")
```

Comment:
FYI, we are planning to remove feature flags and move these into the mesh config : https://github.com/openservicemesh/osm/pull/3536 . You might want to rebase this once the former is merged. 

cc @jaellio

Reply:
are we racing? :eyes:

Reply:
rebased